src/main/resources/application.conf

```diff
@@ -695,14 +695,14 @@ opencomputers {
       # exhaustion, attacking an entity generates 0.3 exhaustion.
       robotExhaustion: 10.0
 
-      # The amount of energy it costs to send a wireless message with signal
-      # strength one, which means the signal reaches one block. This is
-      # scaled up linearly, so for example to send a signal 400 blocks a
+      # The amount of energy it costs to send a wireless message using each tier of
+      # each wireless card with strength one, which means the signal reaches one block.
+      # This is scaled up linearly, so for example to send a signal 400 blocks a
```

Comment:
while I understand the added value in having more configuration options, you're changing the types (in multiple places, here for cost, later for ports, and also range) from an int to an array, and I'll need to test if that causes issues(a crash?) for existing configs. Also, leaving the comment explanation unchanged might be easier to read, if we can and want to use an array for the config value type, perhaps just add a comment after this existing comment block sayingas something like, `# costs for tiers 1 and 2, respectively`

Reply:
I'm on mobile so I can't link it but I added a config patch in Settings.scala for the wireless range and power cost, so it should just update it from older configs (which would mean the value would be reset to default, but this isn't the first time that happened). The old key for ports was changed to only apply to wired, with a new array for wireless.

The main reason why I had to change all that stuff was because nothing was set up to use multiple tiers of wireless cards - everything went off the assumption that there was only 1, hence the arrays replacing single ints/doubles.